browser/src/CopyNumberVariantList/CopyNumberVariantFilterControls.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Checkbox, SearchInput } from '@gnomad/ui'
+
+import CategoryFilterControl from '../CategoryFilterControl'
+
+import { cnvTypes, cnvTypeColors } from './copyNumberVariantTypes'
+
+const CategoryFilterLabel = styled.span`
+  margin-bottom: 0.5em;
+  font-weight: bold;
+`
+
+const CategoryFiltersWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  @media (max-width: 700px) {
+    align-items: center;
+  }
+`
+
+const CheckboxWrapper = styled.div`
```

Comment:
For future reference if you find yourself wanting to write a styled component with no styles, it's probably fine to just use the ordinary HTML element in its place, e.g. `<div>` here. Don't worry about it for now though.